src/components/include/utils/threads/thread.h

```diff
@@ -104,54 +104,45 @@ class Thread : public QObject {
 #else
 class Thread {
 #endif
+
+  typedef enum ThreadCommandEnum {
+    kThreadCommandNone,  // must be first
+    kThreadCommandRun,
+    kThreadCommandStop,
+    kThreadCommandFinalize  // keep last
+    // in case of new commands - update/check threadFunc()
+  } ThreadCommand;
+
+  typedef enum ThreadStateEnum {
+    kThreadStateNone,
```

Comment:
@LuxoftAKutsan, added doxygen comments in [8aacd83](https://github.com/smartdevicelink/sdl_core/pull/707/commits/8aacd83046731bb84d27222048b90d8ebaadf364)